frontend/src/stores/ethers.js

```diff
@@ -65,7 +65,11 @@ export const connectWalletFromLocalStorage = async () => {
 
 export const signMessage = async (msg, signer) => {
   let signedMessage;
-  signedMessage = await signer.signMessage(msg);
+  try{
+    signedMessage = await signer.signMessage(msg);
+  }catch(error){
+    throw error
+  }
```

Comment:
Thanks for wrapping this in a try/catch/throw. I looked at the other functions and this looks properly implemented. I still need to do build/deploy/test locally but I was reviewing this PR and it looks good.

Closing initial review, and continuing on via local testing.